libtransact/src/scheduler/serial/core.rs

```diff
@@ -407,8 +429,11 @@ impl SchedulerCore {
                     sender.send(self.current_batch.take()).map_err(|_| {
                         CoreError::Internal("aborted batch receiver dropped".into())
                     })?;
-                    // Also remove the current transaction
-                    self.current_txn.take();
+
+                    // Also remove the drop the current transaction's context, if it exists
```

Comment:
Typo in comment: Also remove the drop the

Reply:
Fixed